exe/server/main.go

```diff
@@ -43,7 +43,7 @@ func main() {
 	pflag.String("repopath", "~/.powergate", "repo-path")
 	pflag.Bool("embedded", false, "run in embedded ephemeral FIL network")
 	pflag.String("ipfsapiaddr", "/ip4/127.0.0.1/tcp/5001", "ipfs api multiaddr")
-	pflag.Int64("walletinitialfund", 4000000000, "created wallets initial fund in attoFIL")
+	pflag.Int64("walletinitialfund", 4000000000000000, "created wallets initial fund in attoFIL")
```

Comment:
More money per FFS instance, considering that the miner now is more expensive and now we support bigger files.
Tested with a 100mb file (on a `make embed`) and all good. 👍

Reply:
🥳